index.html

```diff
@@ -21,5 +21,22 @@
 		          <img id="drop-btn" alt="drop menu icon" src="./images/dropMenuIcon.png">	   	
 		    </nav>  
     </div>
+	<div id="headline-section">
+	<aside>
```

Comment:
- [ ] It is a good semantic practice to use a `section` tag here instead of a `div` tag. Do well to correct that.